python/delta_sharing/protocol.py

```diff
@@ -160,7 +160,7 @@ class AddFile:
     id: str
     partition_values: Dict[str, str]
     size: int
-    stats: str
+    stats: str = None
```

Comment:
The type hint should be `Optional[str]`?

Reply:
Thank you! I fix it.